File: TP2DPBO2025C2/cpp/Kode Program/petshop.cpp
#include <iostream> 
#include <string>

using namespace std;

class petshop
{
    private:
        string ID;
        string nama_produk;
        string harga_produk;
        int stok_produk;

    public:
        petshop(){
            this->ID = "";
            this->nama_produk = "";
            this->harga_produk = "";
            this->stok_produk = 0;
        }

        petshop(string ID, string nama_produk, string harga_produk, int stok_produk){
            this->ID = ID;
            this->nama_produk = nama_produk;
            this->harga_produk = harga_produk;
            this->stok_produk = stok_produk;
        }

        

        // Getter
        string get_id(){
            return this->ID;
        }

        string get_nama_produk(){
            return this->nama_produk;
        }

        string get_harga_produk(){
            return this->harga_produk;
        }

        int get_stok_produk(){
            return this->stok_produk;
        }

        // Setter
        void set_id(string ID){
            this->ID = ID;
        }

        void set_nama_produk(string nama_produk){
            this->nama_produk = nama_produk;
        }

        void set_harga_produk(string harga){
            this->harga_produk = harga;
        }

        void set_stok_produk(int stok){
            this->stok_produk = stok;
        }

        ~petshop(){

        }
};

File: TP2DPBO2025C2/Java/Petshop.java
public class Petshop {
    
    private String id;
    private String nama_produk;
    private String harga_produk;
    private int stok_produk;

    Petshop(){
        this.id = "";
        this.nama_produk = "";
        this.harga_produk = "";
        this.stok_produk = 0;
    }

    Petshop(String id, String nama_produk, String harga_produk, int stok_produk){
        this.id = id;
        this.nama_produk = nama_produk;
        this.harga_produk = harga_produk;
        this.stok_produk = stok_produk;
    }

    public void setstok_produk(int stok_produk) {
        this.stok_produk = stok_produk;
    }

    public void setid(String id) {
        this.id = id;
    }
    
    public void setharga_produk(String harga_produk) {
        this.harga_produk = harga_produk;
    }

    public void setnama_produk(String nama_produk) {
        this.nama_produk = nama_produk;
    }

    public String getid() {
        return id;
    }

    public String getharga_produk() {
        return harga_produk;
    }

    public String getnama_produk() {
        return nama_produk;
    }

    public int getstok_produk() {
        return stok_produk;
    }

    public void Ingfo(){
        System.out.println("id          : "+ id );
        System.out.println("Nama        : "+ nama_produk);
        System.out.println("Kategori    : "+ harga_produk);
        System.out.println("Harga       : "+ stok_produk);
        System.out.println("//=================//");
    }


}
